remme/clients/basic.py

```diff
@@ -194,31 +193,41 @@ def _socket_request(self, suffix, data=None):
             raise ClientException('Suffix "%s" not supported' % suffix)
 
     def _handle_response(self, msg_type, resp_proto, req):
-        self._stream.wait_for_ready()
-        future = self._stream.send(
-            message_type=msg_type,
-            content=req.SerializeToString())
+        # Loops while NOT_READY
+        while True:
+            self._stream.wait_for_ready()
 
-        resp = resp_proto()
-        try:
-            resp.ParseFromString(future.result().content)
-        except (DecodeError, AttributeError):
-            raise ClientException(
-                'Failed to parse "content" string from validator')
-        except ValidatorConnectionError as vce:
-            LOGGER.error('Error: %s' % vce)
-            raise ClientException(
-                'Failed with ZMQ interaction: {0}'.format(vce))
+            future = self._stream.send(
+                message_type=msg_type,
+                content=req.SerializeToString())
 
-        data = message_to_dict(resp)
+            resp = resp_proto()
 
-        # NOTE: Not all protos have this status
-        with suppress(AttributeError):
-            if resp.status == resp_proto.NO_RESOURCE:
-                raise KeyNotFound("404")
-
-        if resp.status != resp_proto.OK:
-            raise ClientException("Error: %s" % data)
+            try:
+                resp.ParseFromString(future.result().content)
+            except (DecodeError, AttributeError):
+                raise ClientException(
+                    'Failed to parse "content" string from validator')
+            except ValidatorConnectionError as vce:
+                LOGGER.error('Error: %s' % vce)
```

Comment:
Why are you using three different ways to format a string in one pull request? Maintain the consistent code style.

Reply:
Didn't write this code, probably came from merge

Reply:
Anyway please fix it